src/main/java/prgrms/marco/be02marbox/domain/theater/dto/TheaterRoomRequestDto.java

```diff
@@ -0,0 +1,49 @@
+package prgrms.marco.be02marbox.domain.theater.dto;
+
+import java.util.List;
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class TheaterRoomRequestDto {
```

Comment:
```suggestion
public class TheaterRoomRequest {
```
굳이 Dto라는 단어를 붙어야 할까요?